### DISCUSSION TITLE
Total SUM   all pages of grid - SummaryMixIn

### DISCUSSION THREAD

**Discussion by Nikola7i** [serenity-is/Serenity#5346] [Help]
Hi,

I use    CustomSummaryMixin - and Total in footer show only sum of current page.If I have numbers on other 10 pages, how to show total sum ?

Thanks.

**Comment by VictorTomaili** [comment]
For simplest way you can disable pagination if this grid doesn't contains a million records.

**Reply by Nikola7i** [reply]
Tnx. I put more rows per page, solved for sometime. :-)

**Comment by minhhungit** [comment]
There is a solution here, it might help you

https://github.com/serenity-is/Serenity/wiki/Grid---Mixin---Custom-grid-pager-without-counting-total-records
